tests/infrastructure/golden_images/update_boot_source/test_ssp_data_import_crons.py

```diff
@@ -211,12 +211,13 @@ def golden_images_data_import_cron_spec(
 
 @pytest.fixture()
 def created_data_import_cron(
-    unprivileged_client,
+    admin_client,
```

Comment:
could you recheck  data_import_cron_namespace is created by unpriv client so maybe unpriv client works well here .